handlers.go

```diff
@@ -1,120 +1,143 @@
 package main
 
 import (
-	"fmt"
-	"io/ioutil"
-	"net/http"
-    "os"
-    "log"
-    "net/url"
+    "fmt"
     "github.com/gorilla/mux"
     "gopkg.in/mgo.v2/bson"
+    "io/ioutil"
+    "log"
+    "net/http"
+    "net/url"
+    "os"
     "strings"
     "time"
 )
 
-func Index(w http.ResponseWriter, r *http.Request) {
+func Index(w http.ResponseWriter, r * http.Request) {
     w.Header().Set("Content-Type", "text/html; charset=utf-8")
-    b, err := ioutil.ReadFile(PATH_INDEX_PAGE)
+    b, err: = ioutil.ReadFile(PATH_INDEX_PAGE)
```

Comment:
These don't seem to be valid Go syntax. 
@dibyadas You can either use an editor based package (such as [GoSublime](https://github.com/DisposaBoy/GoSublime) for ST3) which handles the formatting for you or use `gofmt -w filename.go`

Reply:
I did `gofmt`. But it used 1 tab as the indent. I was not able to find out how to customise gofmt for our style.  What worked for me was the Indentation settings in Sublime.